src/structures/game_session.ts

```diff
@@ -555,7 +555,21 @@ export default class GameSession {
         if (seekType === SeekType.BEGINNING) {
             seekLocation = 0;
         } else {
-            const songDuration = await getAudioDurationInSeconds(songLocation);
+            let songDuration = (await dbContext.kmq("cached_song_duration")
+                .select(["duration"])
+                .where("vlink", "=", gameRound.videoID)
+                .first())?.duration;
+            if (!songDuration) {
```

Comment:
Modify the `available songs` table creation from`create_kmq_data_tables_procedure.sql` to only include songs that have a location